Misc/NEWS.d/next/Library/2020-05-11-19-17-23.bpo-40597.4SGfgm.rst

```diff
@@ -0,0 +1 @@
+Don't shy away from using Content-Transfer-Encoding for US-ASCII-only emails that have lines over 78 characters long
```

Comment:
If text content lines are longer than policy.max_line_length, always use a content-encoding to make sure they are wrapped.

Reply:
Done. I've also changed the commit title to be more like this. Thanks for the suggestion!